lib/external/io.dart

```diff
@@ -16,7 +16,7 @@ ASTNode funcExit(Runtime runtime, ASTNode self, List<ASTNode> args) {
 
   final exitAST = args[0];
 
-  exit(exitAST.intVal);
+  return IntNode()..intVal = exit(exitAST.intVal);
```

Comment:
Firstly: This is redundant, the program will be exiting so there is nothing you could can by that.

Secondly: Cascade notation would return the original object

Reply:
Should I remove it entirely or correct the IntNode?

Reply:
Remove it entirely